components/tools/OmeroWeb/omeroweb/webgateway/marshal.py

```diff
@@ -158,6 +158,17 @@ def imageMarshal(image, key=None, request=None):
     interpolate = server_settings.get('interpolate_pixels', True)
 
     try:
+        def pixel_size_in_microns(method):
+            try:
+                size = method('MICROMETER')
+                return size.getValue() if size else None
+            except:
+                logger.warn(
```

Comment:
I'd probably think this is `debug` rather than `warn`, though without the typed exception, it's perhaps dangerous.